recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 98e91ccead4d4756ae3c9cde5e09191a8e586d9f4d50838e7ec09d6411dfdb63
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
```

Comment:
There's one more section like this here that could be similarly adapted for consistency:
```
- {{ compiler('c') }}  # [not osx][
    ](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)# Break circular dependency openssl -> clang -> ld64 -> sigtool -> openssl
    - clang_bootstrap_{{ target_platform }} {{ c_compiler_version }}.*  # [osx]
```